File: patterns/rule-deaktivierung/rules-js/_test_dummy_christmas_lights.js
// Reine Test-Hilfsdatei, um "rule-deaktivierung.js" testen zu koennen -
// im echten Einsatz waeren das eure tatsaechlichen Weihnachtslicht-Regeln.
rules.JSRule({
  id: "christmas_lights",
  name: "Christmas Lights (Test-Dummy)",
  triggers: [triggers.ItemCommandTrigger("DummyRuleTrigger")],
  execute: (event) => {
    // absichtlich leer
  }
});

File: patterns/rule-deaktivierung/rules-python/_test_dummy_christmas_lights.py
# Reine Test-Hilfsdatei, um "rule_deaktivierung.py" testen zu koennen -
# im echten Einsatz waeren das eure tatsaechlichen Weihnachtslicht-Regeln.
from openhab import rule
from openhab.triggers import ItemCommandTrigger


@rule(uid="christmas_lights", name="Christmas Lights (Test-Dummy)",
      triggers=[ItemCommandTrigger("DummyRuleTrigger")])
class ChristmasLightsDummy:
    def execute(self, module, input):
        pass
